srtcore/srt_c_api.cpp

```diff
@@ -183,7 +183,7 @@ extern const SRT_MSGCTRL srt_msgctrl_default = {
     PB_SUBSEQUENT,
     0,     // srctime: take "now" time
     -1,    // -1: no seq (0 is a valid seqno!)
-    0,     // 0: no msg/control packet
+    -1,    // -1: unset (0 is control, numbers start from 1)
```

Comment:
This looks like the API change.
Doest the `SRT_MSGCTRL` section of API-functions.md  require an update?

Reply:
No. The documentation doesn't specify what default values are used, only that such a public global object exists.